D8CORE-8019: Updating spacing on Opportunity node right column.

jenbreese opened the pull request:
# READY FOR REVIEW
- @pookmish What do you think the right route is for adding the Contact Us text when the contact fields are required?

# Summary
- fixing up styles for missing right col content 

# Review By (Date)
- next release

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Urgency
- How urgent is this? (Normal, High)

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to...
4. Verify...

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

## Backend / Functional Validation
### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- D8Core-8019](https://stanfordits.atlassian.net/browse/D8CORE-8019)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

jenbreese committed:
D8CORE-8019: adjusting styles

pookmish commented:
@jenbreese What do you mean by "What do you think the right route is for adding the Contact Us text when the contact fields are required?"

jenbreese commented:
Sorry I missed your comment @pookmish. I meant that in the Figma, we have a heading over the contact data. Since Contacts aren't required fields, should the heading be in a template?
<img width="612" alt="Screenshot 2025-05-27 at 8 55 23 AM" src="https://github.com/user-attachments/assets/c6a59da9-c717-493c-92f6-4d6c54e81a3d" />

pookmish commented:
@jenbreese the heading is part of the view configuration in the [stanford_profile PR](https://github.com/SU-SWS/stanford_profile/pull/924). Did you test out the functionality? Is it not working as expected?

pookmish commented:
Oh, @jenbreese I saw what was happening. I've updated the stanford_profile pr with some changes to the view. This should prevent the h2 from displaying if there is no contact info.

jenbreese committed:
adding in other branch changes and working on duplicate lines

jenbreese commented:
Thank you @pookmish. I had gotten myself mixed up since I had two PRs for this Jira. I moved the changes to this PR and deleted the old one. The contact is fixed.  We have a Types/Units block that outputs markup when empty. I didn't see how to make it not do that. Can you show me? 
<img width="328" alt="Types-Units-block" src="https://github.com/user-attachments/assets/ba14c813-b0e4-4186-ac78-4c3ce2be1b6e" />

jenbreese commented:
@pookmish I believe this is ready. I am not longer getting that issue with the empty double lines.

pookmish commented:
@jenbreese remember the PR title request from before. Please use descriptive PR titles.
